src/http.js

```diff
@@ -15,35 +55,26 @@ export const DEFAULT_HEADERS = {
 
 const DEFAULT_OPTIONS = {
   credentials: 'same-origin',
-  csrf: true,
-  headers: DEFAULT_HEADERS,
   mode: 'same-origin',
 }
 
-export default function (endpoint, { root, ...options }) {
+export default function http (endpoint, { root='', csrf=true, ...options }) {
 
   const config = omitUndefined({
     ...DEFAULT_OPTIONS,
+    headers: { ...DEFAULT_HEADERS },
```

Comment:
Sneaky bug here!!!
Before we were using the same headers object for every request, so mutations to it would persist between requests. Yikes! 🙅